### DISCUSSION TITLE
I am creating a UGC wrapper for macOS

### DISCUSSION THREAD

**Discussion by MuntashirAkon** [Eloston/ungoogled-chromium#2082] [Show and tell]
Since UGC does not yet support automated updates, I am creating a wrapper around the Chromium.app on macOS (namely UGC.app) which will be able to check for updates during launch, along with all the other features the Chrome supports (thanks to how apps in macOS works). Visit https://github.com/MuntashirAkon/ugc-config to download and use the app. (To use the app, you only need to put UGC.app at `~/Applications` and open it, you don't need to open Chromium.app.) When most of the goals will be met, a release will be made (so, you can watch for the releases too).

**Goals**
- [x] Opening Chromium without the Apple's annoying scanning notification as well as fixing the removal of the shortcut on every update
- [x] Checking for updates as well as updating the app during launch via **Homebrew** (displaying notification is in progress)
- [ ] Checking for updates after a fixed time interval (via `launchctl`) as well as update the app in the background and issue notification to reopen it when done
- [x] Reading flags from `~/.config/chromium/chromium-flags.conf` just like in Flathub (on Linux).

If you have more ideas, you can share them here or in the [issue tracker](https://github.com/MuntashirAkon/ugc-config/issues/new).

**Comment by henri** [comment]
@MuntashirAkon - thanks for working on the auto-update project. Just curious as to how you got on with this so I can potentially add support for this to a project I work on called [SPB](https://github.com/henri/spb) (AKA Start Private Browser). If your updater is still under active development let me know. Looking at the linked repository it looks like it is not. But it could be that the project is now moved to a different location?
